.distignore

```diff
@@ -0,0 +1,87 @@
+############
```

Comment:
Similar to my questions elsewhere: Why `.distignore` instead of `.gitattributes`?

`.gitattributes` has the benefit that it is also processed by GitHub. When you click "Download ZIP", you only get a ZIP of the actual Performance Lab plugin without the dev ops files. With `.distignore` that is no longer the case.

If we replace `.gitattributes` with `.distignore`, we need a compelling reason why. Can you explain your rationale?

Reply:
Please check my above comment and https://github.com/WordPress/performance/issues/893#issuecomment-1947398713

Reply:
Replied to your reply there :)